galaxy_importer/loaders/collection.py

```diff
@@ -115,6 +116,15 @@ def _check_ansible_test_ignore_files(self):  # pragma: no cover
                 line_count = len(f.readlines())
             self.log.warning(IGNORE_WARNING.format(file=ignore_file, line_count=line_count))
 
+    def _check_collection_changelog(self):
+        """Log an error when a CHANGELOG file is not present in the root of the collection."""
+        CHANGELOG_ERROR = "CHANGELOG.rst file not found at top level of collection."
```

Comment:
There is no tests to see if `CHANGELOG.rst file not found at top level of collection.` gets into the logs (the edits to `tests/unit/test_loader_collection.py` only test missing file logic). I'm not sure if any of the existing unit tests do that which can be used as an example, but we probably should have that here.

Reply:
I am not seeing anything that is exposing the log during the run, everything else is writing directly to the log in the test run then asserting its there. Is there an object or something that is storing the log that we can compare against? I am not seeing this done is the other expected areas.

Reply:
`caplog` can be used for that https://github.com/ansible/galaxy-importer/blob/df274d4021de1314562400c3a4dd4eb657520a13/tests/unit/test_runner.py#L107

Reply:
ah so my understanding of caplog was for use of capturing `pytest` things only. will poke around and see what we can produce.